invenio_communities/communities/services/components.py

```diff
@@ -160,9 +161,9 @@ def _create_set_spec(self, community_slug):
 
     def _create_set_description(self, community_title):
         # NOTE: Does not require translation since this description can also be changed by an admin
-        return "Records belonging to the community '{title}'".format(
-            title=community_title
-        )
+        return _("Records belonging to the community '%(title)s'") % {
+            "title": str(community_title)
```

Comment:
why is it necessary to cast to string?

Reply:
This is beyond the scope of the translation. See: [[GitHub link](https://github.com/inveniosoftware/invenio-communities/blob/daecf7937a5e8a84ce5e6f1d3b897ea8b77d21d1/invenio_communities/communities/services/components.py#L162-L166)].

However, after reviewing the provided note, this autogenerated change is unnecessary. I'll go ahead and revert it.